packages/editor/src/client/i18n.ts

```diff
@@ -1,6 +1,9 @@
 import i18n from 'i18next'
 import {initReactI18next} from 'react-i18next'
 import LanguageDetector from 'i18next-browser-languagedetector'
+import {registerLocale, setDefaultLocale} from 'react-datepicker'
+
+import dateFnsDe from 'date-fns/locale/de'
```

Comment:
Will we be using both date-fns and moment? @penk85 @roosnic1

Reply:
@bishoymelek in #358 @camilleory already removes moment.js